en/docs/reference/config-catalog.md

```diff
@@ -5130,7 +5130,7 @@ log_tracer.enable = true
                 <div class="superfences-content">
                     <div class="mb-config-example">
 <pre><code class="toml">[apim.token.revocation]
-notifier_impl = "org.wso2.carbon.apimgt.keymgt.events.TokenRevocationNotifierImpl"
+notifier_impl = "org.wso2.carbon.apimgt.notification.TokenRevocationNotifier"
```

Comment:
The class needs to be `org.wso2.carbon.apimgt.notification.TokenRevocationNotifierImpl` as per the default.json.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7925/commits/1945f39839794d27ede90b0d7e111d2908c219bb